client/my-sites/plans/jetpack-plans/product-lightbox/style.scss

```
@@ -173,78 +179,60 @@
 	border-radius: 0 8px 8px 0; /* stylelint-disable-line scales/radii */
 }
 
-.product-lightbox__variants-option {
-	margin-top: 1rem;
-	margin-bottom: 2rem;
-	display: flex;
-	flex-direction: row;
+.product-lightbox__variants-options {
+	.form-legend {
+		font-weight: 600;
+		font-size: 0.875rem;
+		margin-bottom: 0.5rem;
+	}
 
-	.multi-checkbox {
+	.form-fieldset {
+		margin-bottom: 0;
+	}
+
+	.multiple-choice-answers-wrapper {
+		margin-top: 1rem;
+		margin-bottom: 1.5rem;
 		display: flex;
+		flex-direction: row;
+	}
 
-		.form-label {
-			padding-right: 1.5rem;
-			span {
-				font-weight: 600;
-			}
-		}
+	.form-label {
+		clear: none;
+		cursor: pointer;
+		display: inline-block;
+		position: relative;
+		padding-right: 3rem;
```

It's better to use [RTL aware](https://hacks.mozilla.org/2015/10/building-rtl-aware-web-apps-websites-part-2/) alternatives:

```suggestion
		padding-inline-end: 3rem;
```